exporter/clickhouseexporter/exporter_traces.go

```diff
@@ -38,6 +38,15 @@ type tracesExporter struct {
 	cfg    *Config
 }
 
+const (
+	otelScopeName    = "otel.scope.name"
+	otelScopeVersion = "otel.scope.version"
```

Comment:
those are defined in collector/semconv as `AttributeOtelScopeName` and `AttributeOtelScopeVersion`. Please use them